server/control/src/main/java/org/kaaproject/kaa/server/control/service/sdk/event/CppEventSourcesGenerator.java

```diff
@@ -163,10 +170,17 @@
                     if (eventFamilyListenersOnGenericEvent.length()>0) {
                         eventFamilyListenersOnGenericEvent += "else ";
                     }
+                    if (eventFamilyListenersNotifyListener.length()>0) {
```

Comment:
please add spaces between eventFamilyListenersNotifyListener.length() and ">" & ">" and "0" respectively